websites/D/Danbooru/presence.ts

```diff
@@ -0,0 +1,74 @@
+const presence = new Presence({
+    clientId: "918794311557058590"
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "danbooru",
+      startTimestamp: browsingTimestamp
+    },
+    shortTitle = document.title.split(/[|]/, 1)[0],
```

Comment:
Use array deconstructing here
```suggestion
    [shortTitle] = document.title.split(/[|]/, 1),
```

Reply:
Done.